pkg/agent/controller/traceflow/packetin.go

```diff
@@ -194,3 +215,15 @@ func getInfoInTunnelDst(regMatch *ofctrl.MatchField) (string, error) {
 	}
 	return regValue.String(), nil
 }
+
+func getInfoInCtNwDstField(matchers *ofctrl.Matchers) (string, error) {
+	match := matchers.GetMatchByName("NXM_NX_CT_NW_DST")
+	if match == nil {
+		return "", nil
+	}
+	regValue, ok := match.GetValue().(net.IP)
+	if !ok {
+		return "", errors.New("packet conntrack IP destination value cannot be got")
```

Comment:
"packet-in conntrack IP destination value cannot be retrieved from metadata"

Reply:
fixed